great-idea/css/index.css

```diff
@@ -63,4 +63,234 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.navigation-bar {
+	display: flex;
+	padding: 10px;
+	justify-content: center;
+	/*width: 850px;
+	top: 0;
+	border-bottom: 0px solid black;
```

Comment:
Don't forget to go back over your old code and remove unwanted code. Have to try to keep it clean for the next person (which may just as well be yourself in the future).